website/source/docs/auth/aws-ec2.html.md

```diff
@@ -1131,9 +1142,11 @@ instance can be allowed to gain in a worst-case scenario.
 <dl class="api">
   <dt>Description</dt>
   <dd>
-   Fetch a token. This endpoint verifies the pkcs#7 signature of the instance identity document.
-   Verifies that the instance is actually in a running state. Cross checks the constraints defined
-   on the role with which the login is being performed.
+Fetch a token. This endpoint verifies the pkcs7 signature of the instance
```

Comment:
Reverted as per the suggestion.
I believed that indenting provided readability but at the same time, it required that only spaces are prefixed. Also, text wrapping gets a little tricky by indenting.